src/dash/parser/matchers/StringMatcher.js

```diff
@@ -32,40 +32,41 @@
  * @classdesc Matches and converts xs:string to string, but only for specific attributes on specific nodes
  */
 import BaseMatcher from './BaseMatcher';
+import DashConstants from '../../constants/DashConstants';
 
 class StringMatcher extends BaseMatcher {
     constructor() {
         super(
             (attr, nodeName) => {
                 const stringAttrsInElements = {
-                    'MPD':                        [ 'id', 'profiles' ],
-                    'Period':                     [ 'id' ],
```

Comment:
In ES6, these keys can also be replaced with DashConstant constants by wrapping them in [ ]. See https://github.com/Dash-Industry-Forum/dash.js/blob/development/src/streaming/XHRLoader.js#L63 for an example.